cleanlab_studio/errors.py

```diff
@@ -24,3 +24,11 @@ def __init__(self) -> None:
 class AuthError(APIError):
     def __init__(self) -> None:
         super().__init__("invalid API key")
+
+
+class InternalError(BaseException):
```

Comment:
```suggestion
class InternalError(Exception):
```

https://docs.python.org/3/library/exceptions.html#module-exceptions

> The built-in exception classes can be subclassed to define new exceptions; programmers are encouraged to derive new exceptions from the [Exception](https://docs.python.org/3/library/exceptions.html#Exception) class or one of its subclasses, and not from [BaseException](https://docs.python.org/3/library/exceptions.html#BaseException). More information on defining exceptions is available in the Python Tutorial under [User-defined Exceptions](https://docs.python.org/3/tutorial/errors.html#tut-userexceptions).